Automattic/simplenote-android
# (WIP) Fix note editor jumping to bottom

### Fix
Should fix #1580 - Saving the y offset of the note editor and restoring it appears to be the cause of the bug. Opening this PR to test a build from. My theory is that on some devices, when the view needs to be adjusted for the soft keyboard, the OS will scroll the view into view and we were _also_ scrolling the view ourselves which would push it to the bottom.

### Test
1. Open a long note
2. Tap in the middle of the note and make some changes
3. The editor scroll position should never change
4. Test making a change on another device to the same note, the editor scroll should stay put
5. Test that search results in a note still scroll into view when you tap the arrows
